mgmt_cli_test.py

```diff
@@ -121,8 +136,22 @@ def create_ks_and_tables(self, num_ks, num_table):
                 # can use this function to populate tables better?
                 # self.populate_data_parallel()
 
-    def verify_backup_success(self):
-        pass
+    # pylint: disable=too-many-arguments
+    def verify_backup_success(self, bucket, cluster_id, keyspace_name='keyspace1', table_name='standard1',
+                              local='/tmp/backup', truncate=True):
+        self.download_file_from_backup_repo(local=local, bucket=bucket, cluster_id=cluster_id)
+        node = self.db_cluster.nodes[0]
+        path_to_tmp = f'{local}/{keyspace_name}/table/{table_name}'
+        table_id = node.remoter.run(f'ls {path_to_tmp}').stdout.strip()
+        path_to_upload = f'/var/lib/scylla/data/{keyspace_name}/{table_name}-{table_id}/upload'
+        node.remoter.run(f'sudo cp {path_to_tmp}/{table_id}/* {path_to_upload}/.')
+        node.remoter.run(f'sudo chown scylla:scylla -Rf {path_to_upload}')
+        if truncate:
+            node.run_cqlsh(f'TRUNCATE {keyspace_name}.{table_name}')
+        node.run_nodetool(sub_cmd='refresh', args=f'-- {keyspace_name} {table_name}')
+        limit = 10
+        after_restore = node.run_cqlsh(f'SELECT * FROM {keyspace_name}.{table_name} LIMIT {limit}').stdout
```

Comment:
a thought maybe using the python driver client would be easier to verify the data. more specifically.
and not just count it.

Reply:
at this point i'm not saving the data before i truncate it... so for now, if it has data, it is already good (although we are not checking for data corruption)